tests/cxx/declare_symbol.h

```diff
@@ -0,0 +1,13 @@
+//===--- declare_symbol.h - test input file for iwyu ----------------------===//
```

Comment:
OK got it. On a related note, what I really wanted to do was group my using decl tests into a directory.  It seems the test runner script supports this. That change would be for another day, but do you have any thoughts on that?